src/Microsoft.WSMan.Management/ConfigProvider.cs

```diff
@@ -2526,9 +2526,9 @@ private string GetHostName(string path)
         private string GetRootNodeName(string ResourceURI)
         {
             string tempuri = string.Empty;
-            if (ResourceURI.Contains("?"))
+            if (ResourceURI.Contains('?'))
```

Comment:
Is this a LINQ usage now? I think it's probably best to keep it as a string

Reply:
But if it's not LINQ, then no issues

Reply:
No, it is String method:
```c#
public bool Contains(char value);
```

Reply:
Ok, looks good